### ISSUE TITLE
Waypoint Visualization (without robot deployment)

### ISSUE THREAD

**Issue by jimpap1** [robodhruv/visualnav-transformer#44]
Hi and sorry for making an issue.
I have trained the model but I don't own a robot so this must be done only with coding.
Can I feed an image from my dataset into the model and get an output on my image with Waypoint Visualization?
Thanks in advance!

**Comment by kejionglee** [comment]
yes you can, just replace the input with opencv instead.

**Comment by jimpap1** [comment]
> yes you can, just replace the input with opencv instead.

How could i do that? In which file should i replace it? Any more help? Thanks!

**Comment by kejionglee** [comment]
deployment/navigate.py,
Chatgpt is your best companion

**Comment by jimpap1** [comment]
> deployment/navigate.py, Chatgpt is your best companion

I managed to get waypoints from navigate.py. Now how do i visualise them on my images?

**Comment by Benaidandan** [comment]
> > deployment/navigate.py, Chatgpt is your best companion
> 
> I managed to get waypoints from navigate.py. Now how do i visualise them on my images?

Did you do that? Does that have to be deployed on the real robot？

**Comment by jimpap1** [comment]
> > > deployment/navigate.py, Chatgpt is your best companion
> > 
> > 
> > I managed to get waypoints from navigate.py. Now how do i visualise them on my images?
> 
> Did you do that? Does that have to be deployed on the real robot？

Yes I got some waypoints from an image dataset without robot.Now i want to visualise these trajectories on the images from the dataset.Using function plot_trajs_on_image needs some arguments that arent computed.

**Comment by caokaifa** [comment]
@jimpap1，Have you solved this problem? I also want to know how this problem was resolved.
